shpc/main/modules/templates/docker.lua

```diff
@@ -34,6 +34,14 @@ For each of the above, you can export:
  - PODMAN_COMMAND_OPTS: to define custom options for the command
 ]]) 
 
+{% if settings.default_version in [False, None] %}
```

Comment:
Hmm. I wonder if there is a way to not have duplication? E.g., right now we have to edit two places to ensure this still works. I haven't looked into it yet but perhaps there is an include syntax we could use?

Reply:
Is `settings.default_version` ro or r+w ? If possible, I would edit it earlier so that only None is used at this stage, not False, and only `module_sys`, not True. Then, this template (and other stuff in `base.py`) can concentrate on the canonical values.

Reply:
The way the settings works, any call to get an attribute will look in self._config[key] so you could reference that dict directly and that will update the settings.default_version.

Reply:
You can also use self.set(key, value) for a nicer UI way to do it!

Reply:
I have made the `Settings` aware of legacy values that are still allowed for backwards compatibility (e.g. False and True for `default_version`) and the new values that are expected instead. It will replace those when loading the config file. Therefore, the rest of shpc (incl. the templates) only need to know about the current preferred values.